nservicebus/msmq/troubleshooting.md

```diff
@@ -77,4 +115,6 @@ While non-transactional messaging in a Network Load Balancing (NLB) environment
  - [Technet content for troubleshooting MSMQ on Windows 2008](https://blogs.msdn.microsoft.com/johnbreakwell/2008/05/07/technet-content-for-troubleshooting-msmq-on-windows-2008-and-vista/)
  - [Publicly available tools for troubleshooting MSMQ problems](https://blogs.msdn.microsoft.com/johnbreakwell/2007/12/13/what-publically-available-tools-are-there-for-troubleshooting-msmq-problems/)
  - [MSMQ service might not send or receive messages after a restart](https://support.microsoft.com/en-us/kb/2554746)
- - [Troubleshooting MSDTC issues with the DTCPing tool](https://blogs.msdn.microsoft.com/distributedservices/2008/11/12/troubleshooting-msdtc-issues-with-the-dtcping-tool/)
\ No newline at end of file
+ - [Troubleshooting MSDTC issues with the DTCPing tool](https://blogs.msdn.microsoft.com/distributedservices/2008/11/12/troubleshooting-msdtc-issues-with-the-dtcping-tool/)
+ - [Where have my MSMQ messages gone?](https://blogs.msdn.microsoft.com/johnbreakwell/2010/01/22/where-have-my-msmq-messages-gone/)
+ - [Test Connectivity Using MQPing](https://technet.microsoft.com/en-us/library/cc731852%28v=ws.10%29.aspx?f=255&MSPPError=-2147217396)
```

Comment:
i think  this link should be https://technet.microsoft.com/en-us/library/cc731852.aspx